mono/metadata/marshal.c

```diff
@@ -149,6 +149,17 @@ init_safe_handle ()
 		mono_defaults.safehandle_class, "DangerousRelease", 0);
 }
 
+G_GNUC_UNUSED static void
+code_for (MonoMethod *method) {
+	MonoMethodHeader *header = mono_method_get_header (method);
+	char *code = mono_disasm_code (0, method, header->code, header->code + header->code_size);
+	char *name = mono_method_full_name (method, TRUE);
+
+	printf ("CODE FOR %s:\n%s\n", name, code);
```

Comment:
the code variable not being freed is a memory leak. but for the name, the verbosity is in fact not necessary